### ISSUE TITLE
Item type selection for reserved PURLs needs to follow the rules for when subtypes should be selected.

### ISSUE THREAD

**Issue by amyehodge** [sul-dlss/happy-heron#1320]
This is currently requiring a subtype in all cases, but should only require subtypes in some cases. 

See details in Design Doc (https://docs.google.com/document/d/1fcr2DYo7OrX-qTdeUOTWKSS1rlq_47sehESmbvgUsrk/edit#heading=h.xatek3rhlho) for rules on subtype selection.

**Comment by jcoyne** [comment]
@amyehodge is this still a problem on stage?  I was unable to reproduce it.

**Comment by amyehodge** [comment]
Yes, I still this issue on stage. In this video I try to assign two different work types that do not require a subtype and instead of taking me to the form, it returns me to the top of the page.

https://images.zenhubusercontent.com/272543337/20e9997b-f47e-498c-984c-8df59c727a10/worktypes.mov

**Comment by jcoyne** [comment]
@amyehodge that image didn't appear to attach.

**Comment by amyehodge** [comment]
@jcoyne It worked briefly because I checked it. Let me try again.


Uploading worktypes.mov…

**Comment by jcoyne** [comment]
If you try to edit a registered PURL, and you select data with no subtypes selected, it kicks you back to the dashboard.   It should accept this as valid selection.  If you did select a subtype, then it takes you to the new version edit page.
